### PR TITLE
Center align Researchers in IRC page

### LINKED ISSUE #715: Center align the word "Researchers" in IRC page
**Is your feature request related to a problem?**
In IRC page we should center the word "Researchers".

**Describe the solution you'd like**
Center the the required topic.

**Additional context**
I'm working on this

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose
The purpose of this PR is to fix #715 

## Goals
To center align topic "Researchers" in IRC web-page.

## Approach
Made the required changes in the code.

### Preview Link
https://pr-717-sef-site.surge.sh/irc.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by anjula-sack — DISMISSED** [review]
Thank You! @Gimhan-minion

**Comment by jayasanka-sack** [comment]
The title is too close to the profile. Could you please increase the top margin of the profile section?

Screenshot 2020-06-15 at 09 40 13

**Review by anjula-sack — DISMISSED** [review]
Thank you! @Gimhan-minion

File: src/main/webapp/irc.html [hunk 1]
```diff
@@ -211,11 +211,9 @@ <h1 class="mb-0 text-onelive font-weight-bold text-center">Frequently Asked Ques
 
 <section class="section pb-0 bg-gradient-white">
     <div class="container">
-        <p class="display-3">Researchers</p>
-
+        <p class="display-3 text-center" style="margin-bottom: 3rem">Researchers</p>
```
Comment (Piumal1999) [inline src/main/webapp/irc.html]: You can use "mb-5" class instead of using inline css

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

File: src/main/webapp/irc.html (hunk 1 above)
Comment (Gimhan-minion) [reply src/main/webapp/irc.html]: Fixed! Thanks @Piumal1999

**Review by Piumal1999 — APPROVED** [review]
Awesome @Gimhan-minion

**Review by anjula-sack — APPROVED** [review]

**Pull request merged**